packages/server/src/cli.ts

```diff
@@ -235,8 +235,13 @@ const bold = (text: string | number): string | number => {
 		} : undefined,
 	});
 
-	logger.info("Starting webserver...", field("host", options.host), field("port", options.port));
-	app.server.listen(options.port, options.host);
+	if (options.noAuth || options.allowHttp) {
+		logger.info("Starting webserver...", field("host", "127.0.0.1"), field("port", options.port));
```

Comment:
We shouldn't adjust the host automatically I feel. Maybe we just output a warning?

Reply:
A warning would also be the easiest option to implement, no?

Reply:
changed it to just a warning